lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -204,6 +210,12 @@ module Resources
             'display': 'Electronic'
           },
           'url': 'http://electronichealth.se/fhir/StructureDefinition/NLLRegistrationBasis'
+        },
+        {
+          'valueCodeableConcept': {
+            'text': 'Does not need pharmacy verification'
+          },
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/pharmacy-verification-status'
```

Comment:
Is this the patient-access block? If so, this won't be returned, right?

Reply:
Good Catch @joeratt this should be removed from patient token workflow.  I will update it.